recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -36,17 +37,8 @@ class GdkPixbufConan(ConanFile):
         "with_introspection": False,
     }
 
-    generators = "pkg_config"
     exports_sources = "patches/**"
```

Comment:
Can you bump the minimum required Conan version to `1.52.0` and use `export_conandata_patches(self)` instead of this?